cmd/commands.go

```diff
@@ -291,6 +291,30 @@ Downloads an updated Caddy binary with the same modules/plugins at the
 latest versions. EXPERIMENTAL: May be changed or removed.`,
 	})
 
+	RegisterCommand(Command{
+		Name:  "add-packages",
```

Comment:
```suggestion
		Name:  "add-package",
```

I think the singular form feels more natural, even if it allows multiple arguments.